ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviderForm.tsx

```diff
@@ -127,6 +127,18 @@ function AuthProviderForm({
                 attributeValue: yup.string().required('Value is a required field'),
             })
         ),
+        claimMappings: yup
+            .array()
+            .of(yup.array().of(yup.string().required('Empty value is not allowed')).length(2))
+            .test('uniqueness test', 'Claim mappings should contain unique keys', (value) => {
+                if (!value) {
```

Comment:
Aha, not to change so close to end of sprint: `!value` here and `value.map(…)` below might be another example of `Array.isArray(value)` that I missed, but on the other side of the coin, it looks like a generic pattern for various data types that you adapted from other validation.